docs/docs/rest-api/public/api/v2/types/volumes.raml

```diff
@@ -7,6 +7,76 @@ uses:
   secrets: secrets.raml
 
 types:
+  CSIVolumeInfo:
+    type: object
+    properties:
+      name:
+        type: string
+        description: A unique identifier for the volume
+      provider:
+        type: string
+        description: The name of the volume provider
+        enum: [ csi ]
```

Comment:
Why do we need a enum which has only as single value?

Reply:
I guess this is to be extendable to more providers in the future? Should we have a `dvdi` enum value also?

Reply:
we introduce a union type of `ExternalVolumeInfo | CSIVolumeInfo` here and use the provider as the tag.

and yes, we also add an according `dvdi`-tag a little down below: https://github.com/mesosphere/marathon/pull/7213/files/0c1833ac26e93c3c73421ebf57582776c49051d4#diff-fc536eff6d0bf5edb192d3e4bb2927e9R94